google/cloud/storage/internal/bucket_metadata_parser.cc

```diff
@@ -189,17 +195,16 @@ std::map<std::string, std::string> ParseLabels(nlohmann::json const& json) {
   return value;
 }
 
-Status ParseProjectNumber(std::int64_t& project_number,
-                          nlohmann::json const& json) {
-  auto p = internal::ParseLongField(json, "projectNumber");
-  if (!p) return std::move(p).status();
-  project_number = *p;
+Status ParseOwner(BucketMetadata& meta, nlohmann::json const& json) {
+  if (!json.contains("owner")) return Status{};
+  Owner o;
+  o.entity = json["owner"].value("entity", "");
+  o.entity_id = json["owner"].value("entityId", "");
```

Comment:
Q.  Are we at all concerned about repeated lookups?

Reply:
I doubt it is in the critical path for anything, but there is no reason to duplicate work either.  Fixed.

Reply:
Without wanting to derail you any further, it seems like we could still go from 2 lookups to 1 using `json.find("owner")` (not that I'm familiar with the API).

Reply:
Tracking that in #9887